avm/src/daemon/daemon.rs

```diff
@@ -94,21 +106,24 @@ async fn post_v1_scale(
 }
 
 // returns cluster delta
-async fn post_v1_stats(cluster_id: &str, deploy_token: &str) -> String {
-  let vm_stats = get_v1_stats().await;
+async fn post_v1_stats(cluster_id: &str, deploy_token: &str) -> Result<String, Box<dyn Error>> {
+  let vm_stats = get_v1_stats().await?;
   let mut stats_body = json!({
     "deployToken": deploy_token,
     "vmStats": vm_stats,
     "clusterId": cluster_id,
   });
-  let cluster_secret = CLUSTER_SECRET.get().unwrap();
+  let cluster_secret = CLUSTER_SECRET.get().unwrap_or_else(|| {
+    error!("No cluster secret");
+    return &None::<String>;
```

Comment:
shouldn't we return an `Err` here?

Reply:
Where it get it's value? If is not possible to get it again for some reason yes, but I thought it was a kind of polling or something that could appear, it does not come from redis?